### PR TITLE
Create a member profile for shubhank-saxena

### LINKED ISSUE #1622: [Community] Member Profile: Shubhank Saxena
**Current Behavior**

@shubhank-saxena  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @shubhank-saxena  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: shubhank-saxena
- Twitter: 19_saxena
- LinkedIn: shubhank-saxena
- Link to profile picture: https://ca.slack-edge.com/TDL4L01AR-U01H9TDUR4P-499a41baa4a2-512

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
Added a member profile for shubhank-saxena

This PR fixes #1622 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/shubhank-saxena/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name: shubhank-saxena
+position: Contributor
+image_path: ./shubhank-saxena.jpeg
+GitHub: shubhank-saxena
+Twitter: 19_saxena
+LinkedIn: shubhank-saxena
+bio: I am an electronics student who is always looking for opportunities to learn something new and grow. I have a goal of helping differentially abled people with my tech and innovation. I have experience with OSS contributions and wearable hardware devices.
+badges: 
+status: Active
+maintainer:
```
Comment (Nikhil-Ladha) [inline src/collections/members/shubhank-saxena/index.mdx:11]: Not required

File: src/collections/members/shubhank-saxena/index.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+position: Contributor
+image_path: ./shubhank-saxena.jpeg
+GitHub: shubhank-saxena
+Twitter: 19_saxena
+LinkedIn: shubhank-saxena
+bio: I am an electronics student who is always looking for opportunities to learn something new and grow. I have a goal of helping differentially abled people with my tech and innovation. I have experience with OSS contributions and wearable hardware devices.
+badges: 
+status: Active
+maintainer: 
+meshmate: false
```
Comment (Nikhil-Ladha) [inline src/collections/members/shubhank-saxena/index.mdx:12]: Not required

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Change the branch to master

**Comment by Anita-ihuman** [comment]
@Nikhil-Ladha  Is this what you meant? I switched the destination to master

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha Is this what you meant? I switched the destination to master

Yes.
Only, the fields which were set to empty/false were not required. Others can be added back.

**Pull request closed without merging**